### DISCUSSION TITLE
Improve documentation of maven internals

### DISCUSSION THREAD

**Discussion by hendrikebbers** [support-and-care/maven-support-and-care#4] [Ideas for Funding]
With the introduction of Maven 4 the documentation of Maven is very outdated. A lot of the internals of Maven have changed. Here the documentation on how Maven internally works should be updated. This documentation will help people to contribute to Maven.

**Comment by hboutemy** [comment]
I'm always worried of these "provide official documentation" issue titles instead of "improve" existing one: one key first step being the identification of what you are looking at and find outdated: https://maven.apache.org/ref/4-LATEST/ ? https://maven.apache.org/guides/development/guide-maven-development.html ? something else?

**Reply by hendrikebbers** [reply]
Good argument. Will change the titles tomorrow

**Comment by ascheman** [comment]
This discussion has been converted to Epic #59 for tracking.

Closing this discussion.
